### PR TITLE
[Community] init tentacles packages

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot/constants.py [hunk 1]
```diff
@@ -65,6 +66,15 @@
 COMMUNITY_TRADINGVIEW_WEBHOOK_BASE_URL = os.getenv(
     "COMMUNITY_TRADINGVIEW_WEBHOOK_BASE_URL", "https://webhook.octobot.cloud/tradingview"
 )
+COMMUNITY_EXTENSIONS_IDENTIFIER = "scaleway"
+COMMUNITY_EXTENSIONS_CHECK_ENDPOINT = os.getenv(
+    "COMMUNITY_EXTENSIONS_CHECK_ENDPOINT", "https://premium.octobot.cloud"
+)
+COMMUNITY_EXTENSIONS_CHECK_ENDPOINT_KEY = os.getenv(
+    "COMMUNITY_EXTENSIONS_CHECK_ENDPOINT_KEY", "TODO"
```
Comment (Herklos) [inline octobot/constants.py:74]: Should we let a "TODO" here?

**Review by Herklos — COMMENTED** [review]
:100:

File: octobot/constants.py (hunk 1 above)
Comment (GuillaumeDSM) [reply octobot/constants.py:74]: yes, we need to add a key in the production function to remove this one

**Review by Herklos — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

File: octobot/octobot.py [hunk 2]
```diff
@@ -233,6 +236,28 @@ async def _ensure_clock(self):
...
+        )
+
+    async def delayed_restart(self, delay):
+        await asyncio.sleep(delay)
+        self.octobot_api.restart_bot()
+
+    async def _on_profile_update(self, profile_name: str):
+        await service_api.send_notification(
+            service_api.create_notification(
+                f"{constants.PROJECT_NAME} will restart in {constants.PROFILE_UPDATE_RESTART_MIN} minutes "
```
Comment (Herklos) [inline octobot/octobot.py:252]: :+1:

**Review by Herklos — APPROVED** [review]

**Pull request merged**